test/app/dftb+/hybrid/periodic/GaAs-kpts-neigh-mic-mpi/dftb_in.hsd

```diff
@@ -29,7 +29,7 @@ Hamiltonian = DFTB {
     }
     Screening = NeighbourBased {
       CutoffReduction = 17.0
-      Threshold = 1e-07
+      Threshold = 1e-100
```

Comment:
Is this not excessively small? Anything below epsilon is probably meaningless.

Reply:
That is true, e.g. 1e-16 would work equally well. I just do not want this particular test to depend on the integral pre-screening scheme. 1e-100 is consistent with the remaining testcases. Any objections?

Reply:
Sure, if it's consistent.